Add rolling friction to contact properties (#3771)
cyberbotics/webots: docs/reference/contactproperties.md

Finally, with four values, both solids use asymmetric coefficients, first two for the first solid and last two for the second solid.
The two friction directions are defined for each faces of the geometric primitives and match with the U and V components used in the texture mapping.
Only the `Box`, `Plane` and `Cylinder` primitives support asymmetric friction.
If another primitive is used, only the first value will be used for symmetric friction.

> The world [asymmetric\_friction1.wbt](../guide/samples-howto.md#asymmetric_friction1-wbt) contains an example of fully asymmetric friction.

Reviewer: We rather use this style although both styles are probably correct:
```suggestion
The [asymmetric\_friction1.wbt](../guide/samples-howto.md#asymmetric_friction1-wbt) world contains an example of fully asymmetric friction.
```
(see other occurrences below)

*Note*: "the readme.txt file" has 1.2M hits in google search while "the file readme.txt" has only 88k.

Finally, with four values, both solids use asymmetric coefficients, first two for the first solid and last two for the second solid.
The two friction directions are defined for each faces of the geometric primitives and match with the U and V components used in the texture mapping.
Only the `Box`, `Plane` and `Cylinder` primitives support asymmetric friction.
If another primitive is used, only the first value will be used for symmetric friction.
WEBOTS\_HOME/projects/sample/howto/asymmetric\_friction/worlds/asymmetric\_friction1.wbt contains an example of fully asymmetric friction.

- The `frictionRotation` allows the user to rotate the friction directions used in case of asymmetric `coulombFriction` and/or asymmetric `forceDependentSlip`.
By default, the directions are the same than the ones used for texture mapping (this can ease defining an asymmetric friction for a textured surface matching the rotation field of the corresponding TextureTransform node).
WEBOTS\_HOME/projects/sample/howto/asymmetric\_friction/worlds/asymmetric\_friction2.wbt illustrates the use of this field.

- The `rollingFriction` field specifies the coefficients of rolling/spinning friction.

> The field can have up to three coefficients, using ODE's nomenclature they are [rho, rho2, rhoN].

Reviewer: These coefficients are mandatory even they are set to 0 or -1.
```suggestion
The field holds three coefficients, using ODE's nomenclature they are [rho, rho2, rhoN].
```

> The field accepts only positive values or -1.0, where -1.0 corresponds to infinity.

Reviewer: ```suggestion
Each coefficient accepts only positive values or -1.0, where -1.0 corresponds to infinity.
```

For a value of zero no rolling friction is applied.
`rho` is the rolling friction coefficient in the first friction direction.
`rho2` is the rolling friction coefficient in the second friction direction, perpendicular to that of `rho`.
`rhoN` is the rolling friction coefficient around the normal direction.

> WEBOTS\_HOME/projects/sample/howto/rolling\_friction/worlds/rolling\_friction.wbt illustrates the effect of the different coefficients.

Reviewer: A hyperlink to the GitHub file would be welcome here (see other places in the documentation where this is done).
